autonomy/cli/helpers/docstring.py

```diff
@@ -66,21 +66,21 @@ def analyse_docstrings(
             App = getattr(module, obj)
             docstring = docstring_abci_app(App)
 
-            original_content = module_path.read_text()
-            has_docstring, exptected_content = compare_docstring_content(
+            original_content = Path(module.__file__).read_text(encoding="utf-8")
+            has_docstring, expected_content = compare_docstring_content(
```

Comment:
tests failed with this because imported module need not be the same file as the `module_path`

https://github.com/valory-xyz/open-autonomy/blob/797361555b3ab4232a7c46b15f122c5a2aea96e6/autonomy/cli/helpers/docstring.py#L63-L69

reason for this failure in [this CI run](https://github.com/valory-xyz/open-autonomy/actions/runs/3566731637/jobs/5993733119)
```
FAILED tests/test_autonomy/test_cli/test_helpers/test_docstrings.py::test_analyse_docstrings_without_update[packages.valory.skills.test_abci]
```